docs/user_guide/branding.rst

```diff
@@ -74,6 +74,22 @@ To reference an external website, make sure your link starts with ``http``:
        }
    }
 
+Customize logo alternative text
+-------------------------------
+
+The logo is by default using "Logo image" as an alternative text of the image. To set
+a specific alt text for this to be accessible to those using screen readers or other
+assistive tech, use the ``html_teme_options["logo"]["alt_text"]`` and provide a new name
+as in the following example:
```

Comment:
```suggestion
You may set a custom ``alt text`` to use with your logo.
This can make the logo more accessible to those using screen readers or other assistive tech.
To do so, use ``html_teme_options["logo"]["alt_text"]`` as in the following example:
```

Reply:
with this modification ,the user loose the fallback value. shouldn't we keep it somewhere ?

Reply:
Yeah I think we could add it back at the end. my reasoning for removing it is that most people probably want to use the docs once they want to change it, not to learn the default.